### ISSUE TITLE
Word level

### ISSUE THREAD

**Issue by tuanchauict** [tmilovanov/wisecreator#16]
Kindle's word wise supports level for words. Is there any way to support in within this script?
Do you know?

**Comment by tmilovanov** [comment]
Yes, it's possible, but right now you need to decide how to handle level for each word by yourself and edit the script in order to do this.

See those lines in main.py:

`query = "CREATE TABLE glosses (start INTEGER PRIMARY KEY, end INTEGER, difficulty INTEGER, sense_id INTEGER, low_confidence BOOLEAN)"`
`query = "INSERT INTO glosses VALUES (?,?,?,?,?)"`
`new_gloss = (start, None, 1, sense_id, 0)` <- 1 here is the level

So third column in glosses table is difficulty.

**Comment by tuanchauict** [comment]
Maybe we can rely on [Google nGrams](https://books.google.com/ngrams). For example, these are words from the "The World is Flat": [armrest,queen,patrons,silk](https://books.google.com/ngrams/graph?content=armrest%2Cqueen%2Cpatrons%2Csilk&year_start=1800&year_end=2000&corpus=15&smoothing=3&share=&direct_url=t1%3B%2Carmrest%3B%2Cc0%3B.t1%3B%2Cqueen%3B%2Cc0%3B.t1%3B%2Cpatrons%3B%2Cc0%3B.t1%3B%2Csilk%3B%2Cc0)

**Comment by peheje** [comment]
This would be awesome. Following.

**Comment by tmilovanov** [comment]
I merged this pull request https://github.com/tmilovanov/wisecreator/pull/25 wordwise now supports word levels. It will be great if someone who concerned commented if this is good choice
